### ISSUE TITLE
test code to check

### ISSUE THREAD

**Issue by Prasadambulkar3** [Prasadambulkar3/test#1]
<!DOCTYPE html>
<html>
<head>
<title>Bootstrap Underline Tab </title>
 <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <link href="http://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet" type="text/css">
</head>

<style>
.tab-container{
  font-family:Montserrat;
  border-radius:10px;
  background:#F4F2CD;
  box-shadow:0px 0px 8px #999;
  padding:30px;
  margin:10px;
  
}

.tabbable-underline > .nav-tabs > li {
  margin-right:20px;
}
.tabbable-underline > .nav-tabs > li > a {
  border:10px;
  margin-right: 0;
  font-weight:bold;
  font-size:24px;
  color: #444;
}

.tabbable-underline > .nav-tabs > li.open, .tabbable-underline > .nav-tabs > li:hover {
  border-bottom: 10px solid #fa4b2a;
  opacity:0.5;
}
.tabbable-underline > .nav-tabs > li.open > a, .tabbable-underline > .nav-tabs > li:hover > a {
  border: 0;
  background: none !important;
  color: #444;
}
.tabbable-underline > .nav-tabs > li.open > a > i, .tabbable-underline > .nav-tabs > li:hover > a > i {
  color: #444;
}
.tabbable-underline > .nav-tabs > li.open .dropdown-menu, .tabbable-underline > .nav-tabs > li:hover .dropdown-menu {
  margin-top: 0px;
}
.tabbable-underline > .nav-tabs > li.active {
  border-bottom: 10px solid #44c764;
  position: relative;
}
.tabbable-underline > .nav-tabs > li.active > a {
  border:10px;
  background:#F4F2CD;
  color:#444;
}

.tabbable-underline > .tab-content {
  margin-top: -9px;
  background-color: #F4F2CD;
  font-style:italic;
  border: 0;
  padding: 30px 0;
}
.portlet .tabbable-underline > .tab-content {
  padding-bottom:10px;
}

.nav-tabs {
border-bottom: 19px solid #fff;
}

.tabbable-underline > .nav-tabs > li {
margin-bottom: -10px;
border-bottom: 10px solid red;
}

.tabbable-underline > .nav-tabs > li {
margin-right: 0px;
}
</style>

<body>

<div class="container">
    <div class="row">
        <div class="col-sm-12">
        
            <div class="tab-container">
                <div class="tabbable-underline">
                    <ul class="nav nav-tabs ">
                        <li class="active">
                            <a href="#tab_no_1" data-toggle="tab">
                            One </a>
                        </li>
                        <li>
                            <a href="#tab_no_2" data-toggle="tab">
                             Two </a>
                        </li>
                        <li>
                            <a href="#tab_no_3" data-toggle="tab">
                            Three </a>
                        </li>
                    </ul>
                    <div class="tab-content">
                        <div class="tab-pane active" id="tab_no_1">
                            <p>
                                Tab No 1
                            </p>
                            <p>
                             Maecenas pellentesque semper ullamcorper. Sed odio turpis, suscipit nectempus sit
                             amet, congue non velit. Duis et lacinia mi. Integer  interdum blandit nunc. Curabitur
                             pretium et felis nec consequat. Aliquam rutrum nisi ac porttitor sollicitudin.
                            <p>
                                <a class="btn btn-success" href="#" target="_blank">
                                    Read More...
                                </a>
                            </p>
                        </div>
                        <div class="tab-pane" id="tab_no_2">
                            <p>
                                Tab No 2
                            </p>
                            <p>
                            Proin viverra nulla vitae nunc pellentesque, et fringilla est tincidunt.
                            Duis enim dui, rhoncus non eros eget, pretium laoreet ipsum. Pellentesque 
                            quis urna nisi. Praesent laoreet malesuada magna vitae suscipit. Donec tempus
                            aliquet lorem eget ullamcorper. Sed non tincidunt metus.Nam ullamcorper vestibulum 
                            blandit. Mauris elit libero, pharetra eu tincidunt sit amet,pellentesque sed nisl.
                            <p>
                                <a class="btn btn-info" href="#" target="_blank">
                                    Read More...
                                </a>
                            </p>
                        </div>
                        <div class="tab-pane" id="tab_no_3">
                            <p>
                                Tab No 3
                            </p>
                            <p>
                            Praesent sagittis sapien ultrices porta accumsan. 
                            Nunc molestie venenatis posuere. Proin rutrum pellentesque nisi, nec finibus augue mattis nec. 
                            <p>
                                <a class="btn btn-warning " href="#" target="_blank">
                                    Read More...
                                </a>
                            </p>
                        </div>
                    </div>
                </div>
            </div>

            
        </div>
    </div>
</div>

<br>
<br>

<br>
<br>

</body>
</html>
